core/src/main/java/org/stellar/anchor/util/ConfigHelper.java

```diff
@@ -0,0 +1,29 @@
+package org.stellar.anchor.util;
+
+import static org.stellar.anchor.util.StringHelper.isEmpty;
+
+import org.stellar.anchor.api.exception.SepValidationException;
+import org.stellar.anchor.config.ClientsConfig;
+import org.stellar.anchor.sep24.Sep24Transaction;
+
+public final class ConfigHelper {
+  public static ClientsConfig.ClientConfig getClientConfig(
+      ClientsConfig clientsConfig, String clientDomain, String sep10Account)
+      throws SepValidationException {
+    ClientsConfig.ClientConfig clientConfig;
+    if (isEmpty(clientDomain)) {
+      clientConfig = clientsConfig.getClientConfigBySigningKey(sep10Account);
+      if (clientConfig != null && clientConfig.getType() == ClientsConfig.ClientType.NONCUSTODIAL) {
+        throw new SepValidationException("Non-custodial clients must specify a client_domain");
+      }
+    } else {
+      clientConfig = clientsConfig.getClientConfigByDomain(sep10Account);
+    }
+    return clientConfig;
+  }
```

Comment:
I know we're just moving this function, but its a bit confusing to me. It seems like we're assuming the account requesting authentication (`sep10Account`) is the same as the `SIGNING_KEY` on a wallet's stellar.toml file. Is that correct? 

@philipliu can you explain how this works?

Reply:
Actually I think I understand, we decided to call accounts that custodial wallets use to authenticate "signing keys".

Reply:
^ I think so

Reply:
Yeah, that's correct. I think this method could benefit from some Javadoc. Do you mind adding it?